tests/test_csr.py

```diff
@@ -0,0 +1,128 @@
+from django.contrib.admin.sites import AdminSite
+from django.http import HttpResponse
+from django.urls import reverse
+
+import pytest
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+
+from simple_certmanager.admin import SigningRequestAdmin, download_csr
+from simple_certmanager.models import SigningRequest
+
+
+@pytest.fixture
+def signing_request():
+    return SigningRequest(common_name="test.com")
+
+
+@pytest.mark.django_db
+def test_admin_create_signing_request(admin_client, db):
+    add_url = reverse("admin:simple_certmanager_signingrequest_add")
+
+    data = {
+        "common_name": "test.com",
+        "country_name": "US",
+        "organization_name": "Test Org",
+        "state_or_province_name": "Test State",
+        "email_address": "test@test.com",
+    }
+
+    response = admin_client.post(add_url, data, follow=True)
+
+    assert response.status_code == 200
+
+    signing_request = SigningRequest.objects.last()
```

Comment:
```suggestion
    signing_request = SigningRequest.objects.get()
```
slightly stronger assertion since it will guarantee only one object is created